src/traces/scattergl/convert.js

```diff
@@ -9,7 +9,7 @@
 
 'use strict';
 
-var createScatter = require('gl-scatter2d');
+var createScatter = require('../../../../gl-scatter2d');
```

Comment:
Can you push your `gl-scatter2d` work to a branch, so that we can work off the same source files?